pkg/obfuscate/sql.go

```diff
@@ -12,10 +12,18 @@ import (
 	"strings"
 	"unicode"
 	"unicode/utf8"
+
+	sqllexer "github.com/DataDog/go-sqllexer"
 )
 
 var questionMark = []byte("?")
 
+// ObfuscationMode valid values
+const (
+	ObfuscateOnly         = "obfuscate_only"
+	ObfuscateAndNormalize = "obfuscate_and_normalize"
+)
```

Comment:
This is cosmetic but a type alias might help people coming back to this in the future to lookup the valid values for `ObfuscationMode`.

```suggestion
type obfuscationMode string

// ObfuscationMode valid values
const (
	ObfuscateOnly         = obfuscationMode("obfuscate_only")
	ObfuscateAndNormalize = obfuscationMode("obfuscate_and_normalize")
)
```